controllers/restic.go

```diff
@@ -137,3 +137,89 @@ func (r *DataMoverBackupReconciler) buildResticSecret(secret *corev1.Secret, dmb
 	secret.Data = resticSecretData.Data
 	return nil
 }
+
+// TODO: move these 2 functions to a common.go and check for DMB or DMR being used
+func (r *DataMoverRestoreReconciler) CreateDMRResticSecret(log logr.Logger) (bool, error) {
+
+	// get datamoverbackup from cluster
+	// TODO: name
+	dmb := pvcv1alpha1.DataMoverBackup{}
+	if err := r.Get(r.Context, types.NamespacedName{Name: "datamoverbackup-sample", Namespace: r.NamespacedName.Namespace}, &dmb); err != nil {
```

Comment:
nit: remove hard-code name value

Reply:
Same as above on waiting until DMB is in backup resources.